src/views/blocks/event-website.php

```diff
@@ -22,13 +23,15 @@
  *
  * @since 5.1.0
  *
- * @param string the target attribute string. Defaults to "_self".
+ * @param string $target The target attribute string. Defaults to "_self".
+ * @param string $href   The link URL.
  */
-$target = apply_filters( 'tribe_get_event_website_link_target', '_self' );
+$target = apply_filters( 'tribe_get_event_website_link_target', '_self', $href, $this->context[ 'post_id' ] );
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: f65c9ae2339dbf681684837cb09a791e</sub>